### PR TITLE
Add prizes inside active challenge

### LINKED ISSUE #592: Display Challenge DF rewards distribution system
Challenge DF rewards are distributed to the first 3 places based on best submission ranking.
Rewards for each place it's fixed and known before so it makes sense for us to make this visible to the users.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #592.

### CODE REVIEW

**Comment by idiom-bytes** [comment]
Looks great but deadline date is wrong.

It should be "Enter Before Aug 03", or "Deadline Aug 02 @ 23:59GMT"

File: src/components/challenges/ActiveCard.svelte [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
   let challengeDetails = {
     bannerTitle: `Challenge-DF ${curEpoch.id}`,
-    challengeDescription: `Crunch some code and participate in the <strong>weekly</strong> data challenge. </br>Predict the price of Ethereum for a share of <strong>${curEpoch.streams[1].substreams[1].rewards} OCEAN</strong> rewards.`,
+    challengeDescription: `Crunch some code and participate in the active data challenge for <strong>predicting the price of Ethereum</strong>. Data Farming challenges are a substream of Ocean Protocol Active Rewards, running over <strong>periods of 1 week</strong>. You can claim your prize from the <a href="https://df.oceandao.org/rewards">Rewards page</a>.`,
     reward: `${curEpoch.streams[1].substreams[1].rewards} OCEAN in Rewards`,
     buttonText: 'PARTICIPATE',
     deadlineText: 'Enter before ' + getUpcomingFirstWednesdayOfTheMonth().format('DD MMM YYYY'),
```
Comment (idiom-bytes) [inline src/components/challenges/ActiveCard.svelte:16]: It's not before this day. It's by EoD. Can we adjust this?

**Review by idiom-bytes — CHANGES_REQUESTED** [review]
Please adjust deadlineText.

File: src/components/challenges/ActiveCard.svelte (hunk 1 above)
Comment (KatunaNorbert) [reply src/components/challenges/ActiveCard.svelte:16]: good point, is there a reason why we are calculating the first Wednesday of the month? I don't think I implemented that one, and this challenge should be weekly.

**Comment by KatunaNorbert** [comment]
I updated the date, I think that was the issue not the text

File: src/components/challenges/ActiveCard.svelte [hunk 2]
```diff
@@ -10,19 +9,27 @@
 
   let challengeDetails = {
     bannerTitle: `Challenge-DF ${curEpoch.id}`,
-    challengeDescription: `Crunch some code and participate in the <strong>weekly</strong> data challenge. </br>Predict the price of Ethereum for a share of <strong>${curEpoch.streams[1].substreams[1].rewards} OCEAN</strong> rewards.`,
+    challengeDescription: `Crunch some code and participate in the active data challenge for <strong>predicting the price of Ethereum</strong>. Data Farming challenges are a substream of Ocean Protocol Active Rewards, running over <strong>periods of 1 week</strong>. You can claim your prize from the <a href="https://df.oceandao.org/rewards">Rewards page</a>.`,
     reward: `${curEpoch.streams[1].substreams[1].rewards} OCEAN in Rewards`,
     buttonText: 'PARTICIPATE',
-    deadlineText: 'Enter before ' + getUpcomingFirstWednesdayOfTheMonth().format('DD MMM YYYY'),
+    deadlineText: 'Enter before ' + moment(curEpoch.date_end).format('DD MMM YYYY'),
```
Comment (idiom-bytes) [inline src/components/challenges/ActiveCard.svelte:15]: Lols.

**Review by idiom-bytes — APPROVED** [review]
LGTM!

**Pull request merged**